frame/staking/src/pallet/mod.rs

```diff
@@ -631,7 +631,7 @@ pub mod pallet {
 				MaxNominatorsCount::<T>::put(x);
 			}
 
-			for &(ref stash, ref controller, balance, ref status) in &self.stakers {
+			for &(ref stash, _, balance, ref status) in &self.stakers {
```

Comment:
Then we don't need `controller` as part of `stakers`?

Reply:
Yes, but that would require companions in Polkadot/Cumulus, and the CI checks are already red.  
So @paritytech/staking-core can do this as follow-up.

Reply:
Or I do it and you force-merge it :laughing:

Reply:
We need prs in Polkadot and cumulus any was to fix ci?

Reply:
I dont think so. The checks in https://github.com/paritytech/cumulus/pull/2574 should go green once this is merged, and unless Polkadot turns red in the companion check here it should be good.

Reply:
Is this change necessary? The controller should be updated in `stakers` to be the same as the `stash`. This is what changed in Substrate in #14039, rather than remove all `controller` args.

Reply:
Amending the `initial_authorities[1]` to be the same a `initial_authorities[0] in https://github.com/paritytech/polkadot/blob/57ec016a97b0e67127bcdf24965ff6bfc16d4cd7/node/service/src/chain_spec.rs#LL404C2-L404C2 for each network will fix this without changing any `controller` arguments?

Reply:
The tests in Cumulus fail without this change, as you can see.

Reply:
@rossbulat is there any follow up on this?

Reply:
I amended 'stakers' here: https://github.com/paritytech/polkadot/pull/7224

Are the cumulus tests working now?

Reply:
No I meant to remove `controllers` from the `Vec` entirely.

Reply:
Issue has been opened.

The most important thing was to put the plug on new unique pairs initially. We now have time to tidy up the code in follow ups.